types/json_ml/json_ml-tests.ts

```diff
@@ -0,0 +1,12 @@
+/// <reference types="node" />
+import { Attributes, Element, Node, parse, stringify } from "json_ml";
+
+const nodes: Node[] = parse("<ul><li>hello</li></ul>");
```

Comment:
The examples on the project homepage might make good test cases: https://github.com/59naga/json-ml

Reply:
The parse example is just "pass a string to parse". I could copy the stringify example but that too is just one example. Any ideas?

Reply:
I see what you mean.  I think the main thing is to include an example of the sort of data that would occur in real usage.  On that basis, I think your current tests are fine.